src/main/scala/beam/analysis/cartraveltime/CarTripStatsFromPathTraversalEventHandler.scala

```diff
@@ -544,15 +544,29 @@ object CarTripStatsFromPathTraversalEventHandler extends LazyLogging {
       val freeFlowTravelTime: Double = calcFreeFlowDuration(freeFlowTravelTimeCalc, linkIds)
       val startCoordWGS = new Coord(driving.startX, driving.startY)
       val endCoordWGS = new Coord(parking.endX, parking.endY)
-      CarTripStat(
-        vehicleId = driving.vehicleId.toString,
-        travelTime = travelTime,
-        distance = length,
-        freeFlowTravelTime = freeFlowTravelTime,
-        departureTime = driving.departureTime,
-        startCoordWGS = startCoordWGS,
-        endCoordWGS = endCoordWGS
-      ) :: acc
+      val outputStats = if (travelTime > 0 & freeFlowTravelTime > 0 & length > 0) {
+        CarTripStat(
+          vehicleId = driving.vehicleId.toString,
+          travelTime = travelTime,
+          distance = length,
+          freeFlowTravelTime = freeFlowTravelTime,
+          departureTime = driving.departureTime,
+          startCoordWGS = startCoordWGS,
+          endCoordWGS = endCoordWGS
+        )
+      } else {
+        logger.warn("Bad path traversals {}", linkIds)
```

Comment:
Should we put the specific metrics that caused this? (`travelTime`, `freeFlowTravelTime`, and `length`)?